scss/components/_activity-feed.scss

```diff
@@ -8,7 +8,7 @@
 
   .civicase__panel-transparent-header {
     margin: auto;
-    width: 830px;
+    width: 870px;
```

Comment:
Can you please remind me what this change do ? Thx in advance.

Reply:
Yes, now that we introduced the Activity Bulk Action Checkbox, the width of the panel holding the activity card and the checkbox needed to be increased.